app/views/teachers/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="login-container">
+<!-- if they're just updating name or email, no additional params -->
+<!-- if they're coming from an email, will have reset in it -->
+<!-- if they're coming from their account page, params will have confirm -->
+
+  <%= form_for current_user, url: {action: "update"} do |f| %>
+    <% @edit_fields.each do |field| %>
+        <% if field.include?("Password") %>
+          <%= f.label field.to_sym %>
```

Comment:
This could be DRYed up a little `<%=f.labe field.to_sym%>` appears in both the `if` and `else`